client/clientBean.go

```diff
@@ -55,6 +55,7 @@ type (
 		GetFrontendClient() frontend.Client
 		GetRemoteAdminClient(cluster string) admin.Client
 		GetRemoteFrontendClient(cluster string) frontend.Client
+		GetRemoteFrontendClients() map[string]frontend.Client
```

Comment:
why introducing `currentClusterName` and `GetRemoteFrontendClients` ?

Reply:
I need to know all the remote frontends. It's a pull model.

Reply:
see my comments in the history initialization

Reply:
Got it.